examples/ex02_cksum/test/ChecksumRTL_test.py

```diff
@@ -54,16 +54,19 @@ def checksum_rtl( words ):
   dut.send.rdy = b1(1)
   while not dut.recv.rdy:
     dut.recv.en = b1(0)
+    dut.eval_combinational()
```

Comment:
Just curious: in what kind of situations would designers want to call `eval_combinational` separately from a `tick`?

Reply:
when you modify some signals and you want to let these changes immediately make effect before calling tick